### PR TITLE
git-xargs 0.0.10 (new formula)

### LINKED ISSUE #43: Homebrew suport
**Is your feature request related to a problem? Please describe.**

Simple install experience via `brew install git-xargs`.

**Describe the solution you'd like**

Using [goreleaser](https://github.com/goreleaser/goreleaser) this should be easy to configure.

Example: https://github.com/estahn/cloudping/blob/master/.goreleaser.yml

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- closes gruntwork-io/git-xargs#43

### CODE REVIEW

File: Formula/git-xargs.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "890e2105ef62117849fdf740d753b6b436b2989a108054913ebeb6b0d93218eb"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.VERSION=v#{version}")
+  end
+
+  test do
```
Comment (carlocab) [inline Formula/git-xargs.rb:14]: Let's check the version here to e to make sure the `ldflags` we use are still valid.

Comment (chenrui333) [reply Formula/git-xargs.rb:14]: Sure

**Review by carlocab — APPROVED** [review]

**Comment by brikis98** [comment]
Nice, thank you @chenrui333!

**Comment by zackproser** [comment]
Thanks @chenrui333!

**Pull request closed without merging**